modules/monitoring-about-specifying-limits-and-requests-for-monitoring-components.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * monitoring/configuring-the-monitoring-stack.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="about-specifying-limits-and-requests-for-monitoring-components_{context}"]
+= About specifying limits and requests for monitoring components
+
+You can configure resource limits and request settings for core platform monitoring components and for the components that monitor user-defined projects, including the following:
+
+* Alertmanager (for core platform monitoring and for user-defined projects)
+* kube-state-metrics
+* monitoring-plugin
+* node-exporter
+* openshift-state-metrics
+* Prometheus (for core platform monitoring and for user-defined projects)
+* Prometheus Adapter
+* Prometheus Operator and its admission webhook service
+* Telemeter Client
+* Thanos Querier
+* Thanos Ruler
+
+When you specify resource limits and requests, you ensure that the containers that run monitoring components have enough CPU and memory resources.
```

Comment:
I think this could be removed, the resource limits and resource requests below already covered it

Reply:
Agree. removed